### ISSUE TITLE
Adding ANSI colors manually seems to break width calculations

### ISSUE THREAD

**Issue by eugenesvk** [pacak/bpaf#401]
Following your tips to manually style via direct injection of ANSI colors from [this discussion](https://github.com/pacak/bpaf/discussions/399#discussioncomment-11385642) I've noticed that I get word-wrapping much earlier

Before: using `literal` style
<img width="629" alt="bpaf-before" src="https://github.com/user-attachments/assets/67e1b74b-a1b5-4bdc-8289-5e891a106981">

After: using `.bold().blue()` style from owo-colors
<img width="629" alt="bpaf-after" src="https://github.com/user-attachments/assets/e827f548-6064-4586-98aa-bfb5b3922b9a">

**Comment by pacak** [comment]
Yeah, right now this would break it. I have a plan to make it more flexible :)
